django/db/models/deletion.py

```diff
@@ -291,7 +293,10 @@ def delete(self):
 
             # fast deletes
             for qs in self.fast_deletes:
-                qs._raw_delete(using=self.using)
+                name = '{0}.{1}'.format(qs.model._meta.app_label, qs.model._meta.object_name)
+                count = qs._raw_delete(using=self.using)
+                if count:
```

Comment:
I think we can omit if statement but zero values will be returned in dict, like this:
```python
>>> Entry.objects.first().delete()
(1, {'webapp.Entry': 1, 'webapp.Entry_authors': 0})
```
What do you think ?